### PR TITLE
Add toggle visibility function and button at login page

### LINKED ISSUE #6: Add password visible toggle button to see password user entered
### Main idea

When users try to login, users can't see password users entered.

스크린샷 2023-07-25 18-12-42

### Alternative ideas

Add visible toggle button at password input.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by gee05053**
This PR resolves https://github.com/lablup/backend.ai-webui-ossca-2023/issues/6.
Users can see password they entered.

스크린샷 2023-07-26 17-37-24
Before PR

offtoggle
After PR off toggle

ontoggle
After PR on toggle

### CODE REVIEW

File: src/components/backend-ai-login.ts [hunk 1]
```diff
@@ -1652,11 +1658,16 @@ export default class BackendAILogin extends BackendAIPage {
...
+                  <div class="horizontal flex layout">
+                    <mwc-textfield type="password" id="id_password" autocomplete="current-password"
+                        label="${_t('login.Password')}" icon="vpn_key"
+                        value="${this.password}"
+                        @keyup="${this._submitIfEnter}">
+                    </mwc-textfield>
+                    <mwc-icon-button-toggle off onIcon="visibility" offIcon="visibility_off"
+                        @click="${(e) => this._togglePasswordVisibility(e.target)}">
+                    </mwc-icon-button-toggle>
+                  </div>
```
Comment (KSoonYo) [inline src/components/backend-ai-login.ts:1672]: I think it would be more natural if the toggle button is placed inside the text field (this is not related to markup, just referring to the styling).

Are you aware of any other methods to style the button?

Comment (gee05053) [reply src/components/backend-ai-login.ts:1672]: I think so too... So I tried to find style at material design, but material design's textField have iconTrailing property but icon button isn't support...

I think same problem at user's info change
스크린샷 2023-07-26 20-09-54

if we use ant design instead of material design, we can apply that toogle button in placed inside the text field!!
ex) [ant design input](https://ant.design/components/input)

Comment (KSoonYo) [reply src/components/backend-ai-login.ts:1672]: well, I guess it's a challenge to convert "material" into "ant". 

I'd like to recommend you to resolve this issue when changing the web component to the react component if you're interested in the task and want to keep managing the issue.

**Review by yomybaby — CHANGES_REQUESTED** [review]
Could you try adjusting the background color? There are too many colors being used. My suggestions are
For the field and icon (eyes, info), use #FAFAFA.
For the top and bottom of the modal, use #FFF.

Here's an image for current:
image

**Comment by gee05053** [comment]
> Could you try adjusting the background color? There are too many colors being used. My suggestions are For the field and icon (eyes, info), use #FAFAFA. For the top and bottom of the modal, use #FFF.
> 
> Here's an image for current: 
image



I modfied background color as your suggestions.
And I tried to modify border bottom line. How about this style??

스크린샷 2023-08-04 오후 4 38 18
